pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/ddp/participant/ParticipantData.java

```diff
@@ -136,10 +133,7 @@ public Map<String, String> getDataMap() {
         if (StringUtils.isBlank(data)) {
             return null;
         }
-        Type type = new TypeToken<HashMap<String, String>>() {
-        }.getType();
-        Gson gson = new Gson();
-        cachedDataMap = gson.fromJson(data, type);
+        cachedDataMap = ObjectMapperSingleton.readValue(data, new TypeReference<Map<String, String>>() {});
```

Comment:
I think this can be simplified to `new TypeReference<>()`

Reply:
Interesting...my intellij suggests it is safe.  Just looking to eliminate redundant explicit typing if it's truly optional.

Reply:
Ah, I get it now. You were not suggesting getting rid of the empty implementation, `{}`.

I can make this change, although this style is prevalent in the code and provides some level of documentation.
Objectively, the second parameter to `readValue` is pretty clumsy, so perhaps just a placeholder is sufficient (rather than attempting to describe the object)?